internal/services/solution_checker_rules/datasource_solution_checker_rules_test.go

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package solution_checker_rules_test
+
+import (
+	"net/http"
+	"os"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/jarcoal/httpmock"
+	"github.com/microsoft/terraform-provider-power-platform/internal/helpers"
+	"github.com/microsoft/terraform-provider-power-platform/internal/mocks"
+)
+
+func TestAccSolutionCheckerRulesDataSource_Basic(t *testing.T) {
+	// Skip unless in acceptance test mode
+	if os.Getenv("TF_ACC") != "1" {
+		t.Skip("Skipping acceptance test")
+	}
+
+	resource.ParallelTest(t, resource.TestCase{
```

Comment:
we don't set parallelism on test with ParallelTest, especially when the test creates environment